docs/src/applications.md

```diff
@@ -45,40 +45,40 @@ DNS is the preferred and recommended discovery method.
 If you deploy your application in the same namespace that contains the
 PostgreSQL cluster, you can also use environment variables to connect to the database.
 
-For example, suppose that your PostgreSQL cluster is called `pg-database`,
-you can use the following environment variables in your applications:
+For example, suppose that your PostgreSQL cluster is called `pg-database`.
+You can use the following environment variables in your applications:
 
-* `PG_DATABASE_R_SERVICE_HOST`: the IP address of the service
-  pointing to all the PostgreSQL instances for read-only workloads
+* `PG_DATABASE_R_SERVICE_HOST` &mdash; The IP address of the service
```

Comment:
I'm a big fan of Unicode. As a result, I'd rather use the actual Unicode character (—) than the SGML entities. See https://www.compart.com/en/unicode/U+2014